db/migrate/20181023152814_add_first_name_and_last_name_to_users.rb

```diff
@@ -0,0 +1,7 @@
+class AddFirstNameAndLastNameToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :first_name, :string
+    add_column :users, :last_name, :string
+    change_column_null :users, :first_name, :last_name, false
```

Comment:
This migration if cool if you are doing a new migration, if you are just changing the original migration i would prefer you to add `null: false` in the `add_column` declaration. It's easy to read